### PR TITLE
#162633456 setup SASS

### PR DESCRIPTION
**Pull request by NabunyaLilian**
**What does this PR do?**

Sets up SASS processing on client.

**Description of Task to be completed?**

- Install loaders; sass-loaders, sass-node loaders, css-loaders and style-loaders. 
- Update the webpack.config.js file with configurations for SASS and CSS.
- Create css and sass files.

**How should this be manually tested?**
- Run `yarn` to install all the dependencies.
- Run `yarn start` 

**What are the relevant pivotal tracker stories?**

[#162633456](https://www.pivotaltracker.com/story/show/162633456)

### CODE REVIEW

**Comment by jeanjoe** [comment]
When i run `yarn start` the build is not successful, it complains of your new modules not being found. Please update the PR description to first install your new modules before `yarn start`.

**Comment by mubstimor** [comment]
LGTM

File: src/assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: #d9d9d9;
```
Comment (mubstimor) [inline src/assets/css/style.css:2]: I suggest that short css definitions like this one could be written on one line e.g body{xx:cc;}
Makes the files less cluttered, the bigger it gets.

Comment (fatukunda) [reply src/assets/css/style.css:2]: @mubstimor I think what Lillian did is the right convention in my understanding.

**Comment by fatukunda** [comment]
LGTM

**Comment by NabunyaLilian** [comment]
@jeanjoe , thanks for the feedback let me work on it.

**Comment by mycok** [comment]
LGTM

**Pull request merged**